src/index.js

```diff
@@ -106,29 +87,28 @@ const component = () => {
       const description = document.createElement('textarea');
       description.className = 'description';
       description.rows = 'auto';
-      description.value = todo.description.toLowerCase().charAt(0).toUpperCase();
-      description.value += todo.description.slice(1);
+      description.value = capitalize(todo.description);
       element.appendChild(description);
 
       const taskButton = document.createElement('button');
       taskButton.className = 'task-button';
       taskButton.innerHTML = '<i class=\'ellipsis vertical icon\'></i>';
       element.appendChild(taskButton);
 
-      // TODO: This code is reserved for the next project task.
-      // description.addEventListener('focus', (e) => {
-      //   e.target.parentNode.style.background = '#ffff0080';
-      //   e.target.parentNode.lastElementChild.innerHTML = `<i class="trash alternate icon"></i>`;
-      // });
-
       // Handle checkbox change event
-      checkboxEvent(checkbox, todo, saveToLocalStorage, refreshPage);
+      checkboxEvent(checkbox, todo, todoListData, saveToLocalStorage, refreshPage);
       isComplete(todo.completed, description);
 
       todoContainer.appendChild(element);
     });
   }
 
+  // Update active item
+  updateList(todoListData, saveToLocalStorage, refreshPage);
+
+  // Retrieve parameters for delete button
+  deleteItem(todoListData, createIndexes, saveToLocalStorage, refreshPage);
```

Comment:
- [ ] The icon to **delete** an item is missing in your application's User Interface, as otherwise can be seen in this [video of the UI desgin](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Kindly ensure to include the _delete_ or _trash icon_ as can be seen in the above video, and hook the delete method to it accordingly.

Reply:
Hi, @rloterh
Thanks for noticing the index value issue when adding new items. Changes have been made accordingly.

However, about the second comment. I noticed some irregularities with how the functionality of the ellipses looks like on the video versus how it actually works on the minimalist website. On the video, it appears like they are clicking on the ellipses icon to reveal the delete button. But on the website clicking on the ellipses does nothing since it is strictly reserved for dragging and dropping. The delete button as well as the yellow background appears when you click anywhere on the text area for editing purposes. I applied the same functionality on my website. If you attempt to edit by clicking anywhere around the text area, the delete button will appear.

Moreover, If I were to attempt to apply a click event on the ellipses to reveal the delete button as you specified. It would make it very difficult if not nearly impossible to also apply 'drag' and 'drop' events at the same time. Please kindly test my query on the [minimalist website](https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/) to witness my point. 

Thanks for the great review.

Reply:
Hello @KDlamini 

You are right. I've verified and noticed that you've implemented the said functionality. I apologize that I missed this point in my review. Cheers and Happy coding

Reply:
**For next reviewer**

Kindly consider my last review comments made on **missing delate button** as invalid, as I mistakingly missed out on steps that were actually implemented on this project. Thanks in anticipation.